source/metadata.py

```diff
@@ -11,10 +11,18 @@
 # This is used by automation like the Pinecil updater
 
 
-if len(sys.argv) != 2:
+if len(sys.argv) < 2:
     print("Requires the output json name as an arg")
     exit(1)
 
+# Explicitly provide target file to scan for json output
+ModelName = None
+if len(sys.argv) == 3 and sys.argv[2] is not None and sys.argv[2] != "":
+    ModelName = sys.argv[2]
+    if ModelName == "Pinecil_multi-lang" or ModelName == "Pinecilv2_multi-lang":
```

Comment:
Probably better to search for `_multi-lang` instead as we may add this feature to TS101

Reply:
Nice future-proof suggestion. Didn't know about possible TS101/multilang case. Will test a couple of modifications here before updating PR.